### PR TITLE
format link generation as module, defined all parameters

### PR DESCRIPTION
**Pull request by jwong-nd**
moving scripting code into a module for calling inside a code capsule

### CODE REVIEW

File: src/ng_link/scripts/create_links.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import dataset_links
+
+if __name__ == '__main__':
```
Comment (camilolaiton) [inline src/ng_link/scripts/create_links.py]: It's recommended to wrap the main inside of a main function

File: src/ng_link/dataset_links/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+"""
+Init file of utility functions
+"""
+__version__ = "0.0.1"
```
Comment (camilolaiton) [inline src/ng_link/dataset_links/__init__.py]: Is it necessary to have another module inside of scripts? If it's completely necessary, then there's no problem for now but we have to think if it's worth moving this inside of `ng_link`.

**Review by camilolaiton — CHANGES_REQUESTED** [review]
Hey Jonathan, amazing scripts. Could you please run the linters and fix what's necessary before I approve this? Thanks!

**Comment by jwong-nd** [comment]
Not sure what the interrogate issue is, all functions have docstrings

File: pyproject.toml [hunk 3]
```diff
@@ -22,7 +22,8 @@ dependencies = [
     'sphinx',
     'pint',
     'numpy',
-    'neuroglancer'
+    'neuroglancer',
```
Comment (camilolaiton) [inline pyproject.toml]: Can we pin `numpy`, `neuroglancer`, `pint` and `xmltodict` dependencies?

**Review by camilolaiton — CHANGES_REQUESTED** [review]

**Review by camilolaiton — APPROVED** [review]

**Pull request merged**